lib/elixir/test/elixir/code_test.exs

```diff
@@ -95,7 +95,7 @@ defmodule CodeTest do
       sample = """
       defmodule CodeTest.UnknownRemoteCall do
         def perform do
-          UnkownModule.foo()
+          UnknownModule.foo()
```

Comment:
Please update the assertion below to match on position {3, 19}, as it checks for the position of the dot. :)